### PR TITLE
[SPLIT #221]  File structure changed & styles for new components

### PR DESCRIPTION
**Pull request by 18601673727**

### CODE REVIEW

File: app/store/configureStore.js [hunk 1]
```diff
@@ -1,36 +0,0 @@
-import { createStore, applyMiddleware, compose } from 'redux';
-import { routerMiddleware } from 'react-router-redux';
-import thunk from 'redux-thunk';
-import rootReducer from 'reducers';
-import promiseMiddleware from 'middlewares/promiseMiddleware';
-import createLogger from 'redux-logger';
-
-/*
- * @param {Object} initial state to bootstrap our stores with for server-side rendering
- * @param {History Object} a history object. We use `createMemoryHistory` for server-side rendering,
```
Comment (choonkending) [inline app/store/configureStore.js]: Was this accidentally removed from this PR? Hehe

Comment (18601673727) [inline app/store/configureStore.js]: Nope, new configureStore.js is at `app/` folder

Comment (choonkending) [inline app/store/configureStore.js]: Hey I just had a look again. https://github.com/18601673727/react-webpack-node/tree/348472fb49c5fa402bb7764a6de45b3f443b508c/app

I think the `configureStore` was added in a separate commit.

**Comment by choonkending** [comment]
I think the changes in this PR are good. 

Only comment is that, perhaps we should retain having the `index.js` file in the `types` folder, but that could be a separate fix. Really sorry, given that there has been a lot of moving changes over the past few weeks, could you merge master and push this change?

I'll merge this once it's resolved. Otherwise, if you have no capacity to change this, I'll apply your fixes as a separate commit if you like.

**Comment by choonkending** [comment]
Hey buddy, I think the recent changes [here](https://github.com/choonkending/react-webpack-node/blob/master/app/actions/messages.js#L2) I've made might make this some of these files error out, because they expect to import from `types` and the file no longer exists.

Would you like me to push fixes to your fork or you could resolve them?

**Comment by 18601673727** [comment]
@choonkending You'll need to merge https://github.com/choonkending/react-webpack-node/pull/269 in before this one.

**Comment by choonkending** [comment]
I will close this PR for now - as unfortunately the code we have has changed significantly from when the time this PR was open. We'll definitely use the snippets in this PR (where relevant). Thanks for spending the time to create this patch!

**Pull request closed without merging**